hello-world/demo.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * @license
+ * Copyright (c) 2017 Google Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+const {
+  BrowserLoader,
+  SlotComposer,
+  DemoBase,
+  Arc,
+  systemParticles,
+  workerPecFactory,
+} = window.Arcs;
+
+function ContextFactory({ loader, pecFactory, slotComposer }) {
```

Comment:
ContextFactory doesn't have a particular domain meaning related to Arcs. Instead, it is more of a helper that is useful when making demos. Is this correct? If so, it might be nice to call this out.